benchmark/stats/histogram.go

```diff
@@ -126,12 +123,14 @@ func NewHistogram(opts HistogramOptions) *Histogram {
 		sumOfSquares: newCounter(),
 		tracker:      newTracker(),
 	}
-	low := opts.MinValue
-	delta := opts.SmallestBucketSize
+	delta := 1.0
 	for i := 0; i < opts.NumBuckets; i++ {
```

Comment:
delta := 1.0
h.buckets[0].lowBound = ...
for i := 1; i < opts.NumBuckets; i++ {
 ...
}